tests/test_entities.py

```diff
@@ -1530,6 +1540,19 @@ def test_usergroup_with_external_usergroup(self):
         self.assertTrue(hasattr(usergrp, 'usergroup'))
         self.assertIn('usergroup', read.call_args[0][2])
 
+    def test_subnet(self):
+        """Call :meth:`nailgun.entities.Subnet.read`.
+
+        Ensure that the ``from_`` attribute is successfully set.
```

Comment:
This statement isn't covered, that was probably a bad advice to copy `test_discovery_rule` which doesn't verify it as well, sorry for that.
Just add one extra line in the end of test:
```python
self.assertIn('from_', read.call_args[0][1])
```

Reply:
oops, i actually had that in there, but screwed up the commiting part :)

Reply:
done